eng/common/msbuild.sh

```diff
@@ -54,5 +54,6 @@ if [[ "$ci" == true ]]; then
   node_reuse=false
 fi
 
+echo "MSBuild $extra_args"
```

Comment:
This change will be lost when we pull another Arcade update.

Is this a local-debugging-only change?
```suggestion
```

Reply:
Oh, sorry about that one, that was indeed a temporary instrumentation helping me nail down some of the build issues, I'll revert this part of the change.